regipy/plugins/ntuser/shellbags_ntuser.py

```diff
@@ -3,18 +3,406 @@
 from regipy.hive_types import NTUSER_HIVE_TYPE
 from regipy.plugins.plugin import Plugin
 from regipy.utils import convert_wintime
+from plaso.helpers.windows import shell_folders
```

Comment:
This is a really large dependency.  Is there any way to do this without it?

Reply:
Done